serverless.yml

```diff
@@ -23,14 +23,16 @@ frameworkVersion: '2'
 provider:
   name: aws
   runtime: nodejs12.x
+  stage: dev
+  region: eu-west-1
 
 plugins:
   - serverless-finch
   - serverless-single-page-app-plugin
 
 custom:
   client:
-    bucketName: partyshe-second-app
+    bucketName: partyshe-third-app
```

Comment:
Bucket should be the same

Reply:
Thank you, Sergey. I understand. I've deleted all services and buckets and played with deployment. 
If naming is not critical and if you don't mind, I want to keep current version.